build.cmd

```diff
@@ -0,0 +1,14 @@
+@echo off
```

Comment:
I'm fine with keeping this file, but I actually believe PS is much more usable in every respect. Even the ExecutionPolicy error is explained when it happens, IIRC.

Anyway, as long as it doesn't have to grow, let's keep the .cmd file.